CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,135 @@
+* Any extra context you believe is relevant for the incident
+* If you believe this incident is ongoing
+* Any other information you believe we should have
+
+*Note*: with anonymous reports, there are inherent limits in our ability to do follow-up or collect additional information. If you are uncomfortable sharing your identity, consider creating a throw-away email address without your real name so that we will be able to reach you.
+
+All complaints will be reviewed and investigated promptly and fairly. All Code of Conduct committee members are obligated to respect the privacy and security of the reporter of any incident. Further details of committee membership policies may be posted separately. 
+
+## Enforcement Guidelines
+
+Community leaders will follow these Community Impact Guidelines in determining
+the consequences for any action they deem in violation of this Code of Conduct:
+
+### 1. Correction
+
+**Community Impact**: Use of inappropriate language or other behavior deemed
+unprofessional or unwelcome in the community.
+
+**Consequence**: A private, written warning from community leaders, providing
+clarity around the nature of the violation and an explanation of why the
+behavior was inappropriate. A public apology may be requested.
+
+### 2. Warning
+
+**Community Impact**: A violation through a single incident or series of
+actions.
+
+**Consequence**: A warning with consequences for continued behavior. No
+interaction with the people involved, including unsolicited interaction with
+those enforcing the Code of Conduct, for a specified period of time. This
+includes avoiding interactions in community spaces as well as external channels
+like social media. Violating these terms may lead to a temporary or permanent
+ban.
+
+### 3. Temporary Ban
+
+**Community Impact**: A serious violation of community standards, including
+sustained inappropriate behavior.
+
+**Consequence**: A temporary ban from any sort of interaction or public
+communication with the community for a specified period of time. No public or
+private interaction with the people involved, including unsolicited interaction
+with those enforcing the Code of Conduct, is allowed during this period.
+Violating these terms may lead to a permanent ban.
+
+### 4. Permanent Ban
+
+**Community Impact**: Demonstrating a pattern of violation of community
+standards, including sustained inappropriate behavior, harassment of an
+individual, or aggression toward or disparagement of classes of individuals.
+
+**Consequence**: A permanent ban from any sort of public interaction within the
+community.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org/version/2/1/code_of_conduct.html)
+version 2.1.
+
+Sections of this text also benefitted from [Django](https://www.djangoproject.com/conduct/reporting/) and [Jupyter](https://jupyter.org/governance/conduct/reporting_online.html) reporting guidelines. 
+
```

Comment:
I see some other projects include a Diversity Statement in the Code of Conduct to encourage contributors from diverse backgrounds. Do we need that and do we have that somewhere?

Reply:
yes! just putting this here as a reminder of how conda does it https://github.com/conda-incubator/governance/blob/main/CODE_OF_CONDUCT.md#conda-organization-diversity-statement